app/javascript/packs/ping_things/index_template.vue

```diff
@@ -20,7 +20,7 @@
       </div>
     </div>
 
-    <ping-thing-table :ping_things="ping_things_for_table()" :network="network" />
+    <ping-thing-table :network="network" />
```

Comment:
:ping_things could be there at least to grab initial data - there would be no need to request for pings twice on page load